djangocms_blog/cms_menus.py

```diff
@@ -43,9 +47,13 @@ def get_nodes(self, request):
         config = False
         if self.instance:
             if not self._config.get(self.instance.application_namespace, False):
-                self._config[self.instance.application_namespace] = BlogConfig.objects.get(
-                    namespace=self.instance.application_namespace
-                )
+                try:
+                    self._config[self.instance.application_namespace] = BlogConfig.objects.get(
+                        namespace=self.instance.application_namespace
+                    )
+                except Exception as e:
```

Comment:
IDK if it's work to have a stricter check here (`BlogConfig.DoesNotExist` and `AttributeError` should suffice) or keep it broad :thinking:

Reply:
Totally agree on this one.